cfgov/v1/models/learn_page.py

```diff
@@ -301,20 +303,44 @@ class EventPage(AbstractFilterPage):
     def page_js(self):
         return super(EventPage, self).page_js + ['video-player.js']
 
+    def get_venue_coords(self):
+        # Default to Washington DC coordinates
+        venue_coords = '-77.039628,38.898238'
+
+        if not self.venue_city or not self.venue_state:
+            return venue_coords
+
+        location = '{} {}'.format(self.venue_city, self.venue_state)
+        api = 'https://api.mapbox.com/geocoding/v5/mapbox.places-permanent/'
+        location_api_url = api + location + '.json'
+
+        params = {'access_token': settings.MAPBOX_ACCESS_TOKEN}
```

Comment:
Should we test here to see if `MAPBOX_ACCESS_TOKEN` is defined before we send the request?

Reply:
Yesh, I'll check for it on L310.